frontend/src/metabase/lib/entities.js

```diff
@@ -661,3 +665,17 @@ export const notify = (opts = {}, subject, verb) =>
 
 export const undo = (opts = {}, subject, verb) =>
   merge({ notify: { subject, verb, undo: true } }, opts || {});
+
+// makes entity framework-compatible api requests
+export async function rtkApiRequest(dispatch, endpoint, input) {
+  const action = dispatch(
+    endpoint.initiate(input, {
+      forceRefetch: true,
```

Comment:
The assumption here is that the request state is still managed by the framework. If it reaches this point, we need to bypass the RTK cache.